ui/app/utils/identity-manager.js

```diff
@@ -0,0 +1,47 @@
+// manage a set of unique ids
```

Comment:
Moved this to a utility so it could be used not only for mirage but in a case where we need to generate ids for relationships or anything else.